staging/src/k8s.io/client-go/tools/clientcmd/overrides.go

```diff
@@ -100,6 +102,19 @@ func (f FlagInfo) BindStringFlag(flags *pflag.FlagSet, target *string) FlagInfo
 	return f
 }
 
+// BindStringSliceFlag binds the flag based on the provided info.  If LongName == "", nothing is registered
+func (f FlagInfo) BindStringArrayFlag(flags *pflag.FlagSet, target *[]string) FlagInfo {
+	// you can't register a flag without a long name
+	if len(f.LongName) > 0 {
+		sliceVal := []string{}
+		if len(f.Default) > 0 {
+			sliceVal = strings.Split(f.Default, " ")
```

Comment:
can't assume `" "` is the delimiter... either don't support multi-value defaults or allow setting a slice as the default somehow